Fix minor issues in Developer and Contributor docs (#74)
jupyter-server/gateway_provisioners: docs/source/developers/custom-images.md

## Extending Existing Kernel Images

A common form of customization occurs when the existing kernel image is serving the fundamentals but

> the user wishes to extend the image with additional libraries to prevent the need to import them within
> the Notebook interactions. Since the image already meets the

Reviewer: I think this wording was improper originally.  The idea behind adding the libraries to the image is to prevent their _installation_ during notebook interactions.  They need to be imported regardless.  So perhaps something like:
```suggestion
the user wishes to extend the image with additional libraries to prevent the need to install them during
the Notebook's execution. Since the image already meets the
```

Author: Yeah I figured that was sort of what this sentence was getting at but didn't want to completely overhaul it. Your suggestion makes it perfectly clear, though!
